azure/iac/sql_servers.rego

```diff
@@ -16,10 +23,19 @@ azure_issue["sql_public_access"] {
 
 sql_public_access {
```

Comment:
Rule name should be `sql_public_access_disabled`

Reply:
fixed